src/app/public/modules/code/code.component.scss

```diff
@@ -1,24 +1,24 @@
 @import '~@skyux/theme/scss/mixins';
 
 .sky-code {
+  @include sky-border("dark", "top", "right", "bottom", "left");
```

Comment:
What if we only reverted the text color to red, but kept the newer background color and removed the border? I mostly dislike the border. @Blackbaud-AdamFunderburk?

Reply:
The background color with red text doesn't show enough, (not accessibly compliant). I prefer the border personally, I find the background color without the border to be more straining on the eyes. Did remove the rounded corners to meet our style guidelines though.

Reply:
Just my 2 cents, but I agree with Brandon on the border.

Reply:
You're right - the red text on that background color doesn't pass accessibility contrast requirements. I iterated with Matt Gregg and we agreed that lightening/removing the background color was a **_worse_** answer than darkening the red text for the overall impact. It's currently using `$sky-background-color-danger-dark` which is #d93a3d. Darkening the text by 5% to #d1282c puts us in One-Off Land, but leaves us with a much better result.

Good call on removing the border radius.

Reply:
👍 @Blackbaud-AdamFunderburk I added the 5% darken to the text.